### ISSUE TITLE
Bug in pagination due to accessors array position

### ISSUE THREAD

**Issue by Jimexist** [searchkit/searchkit#11]
When `PageSizeAccessor` appears after the `PaginationAccessor`, the latter will pick up `20` as the page size (because it's `0` before `PageSizeAccessor` steps in) instead of what's specified in `PageSizeAccessor`.

In Chrome console, at stack trace:
```
PaginationAccessor.buildOwnQuery (PaginationAccessor.js:20)
(anonymous function) (Searcher.js:30)
arrayReduce (index.js:1450)
(anonymous function) (index.js:3445)
Searcher.buildQuery (Searcher.js:29)
(anonymous function) (SearcherCollection.js:40)
arrayEach (index.js:1289)
(anonymous function) (index.js:3345)
SearcherCollection.buildQuery (SearcherCollection.js:40)
SearchkitManager.buildQuery (SearchkitManager.js:55)
SearchkitManager._search (SearchkitManager.js:97)
SearchkitManager.performSearch (SearchkitManager.js:85)
Pagination.setPage (Pagination.js:48)
React.cloneElement.onMouseDown (FastClick.js:17)
ReactErrorUtils.invokeGuardedCallback (ReactErrorUtils.js:71)
executeDispatch (EventPluginUtils.js:79)
executeDispatchesInOrder (EventPluginUtils.js:102)
executeDispatchesAndRelease (EventPluginHub.js:43)
executeDispatchesAndReleaseTopLevel (EventPluginHub.js:54)
forEachAccumulated (forEachAccumulated.js:23)
EventPluginHub.processEventQueue (EventPluginHub.js:259)
runEventQueueInBatch (ReactEventEmitterMixin.js:18)
ReactEventEmitterMixin.handleTopLevel (ReactEventEmitterMixin.js:34)
handleTopLevelWithoutPath (ReactEventListener.js:93)
handleTopLevelImpl (ReactEventListener.js:73)
Mixin.perform (Transaction.js:136)
ReactDefaultBatchingStrategy.batchedUpdates (ReactDefaultBat…Strategy.js:62)
batchedUpdates (ReactUpdates.js:94)
ReactEventListener.dispatchEvent (ReactEventListener.js:204)
```
```
> this.accessors
> [SearchAccessor, SortingAccessor, FacetAccessor, FacetAccessor, FacetAccessor, FacetAccessor, NumericOptionsAccessor, FacetAccessor, PaginationAccessor, PageSizeAccessor]
```

and also at that point

```
> _.reduce(this.accessors, function(query, accessor) { console.log('query page size', query.query.size);return accessor.buildOwnQuery(query);}, query)
```
gives:
```
(10 times) query page size 0
```

**Comment by ssetem** [comment]
thanks, we will fix soon in upcoming release

**Comment by ssetem** [comment]
ok this is now fixed on our develop branch

https://github.com/searchkit/searchkit/blob/e4d9b0df671a41e288549d7dc1a333140d99d51d/src/__test__/core/accessors/PaginationAccessorSpec.ts#L42

Thanks again!

**Comment by Jimexist** [comment]
Thanks for the quick fix!

**Comment by ssetem** [comment]
released now in 0.3.2
